propagators/jaeger/jaeger_data_test.go

```diff
@@ -162,12 +162,6 @@ var invalidExtractHeaders = []extractTest{
 			jaegerHeader: fmt.Sprintf("%s:%s:0:1", traceID32Str+"0000", spanIDStr),
 		},
 	},
-	{
-		name: "trace ID length is not 32 or 16",
```

Comment:
Please replace this with a valid test.

Reply:
There is another test `trace ID length > 32` and the propagator now accepts all shorter trace IDs that were previously invalid (ignoring the special case for an empty one that's handled differently).

I've added a test for a 15 char hex string trace ID.